regen-network/regen-web
# feat: add email address and social login options to login popup

## Description

Closes: #2154
Needs for the google login to work: https://github.com/regen-network/regen-server/pull/400

Global renaming of WalletButton/Modal to LoginButton/Modal



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



1. Test login modal in Storybook: https://deploy-preview-2172--regen-storybook.netlify.app/?path=/story/registry-organisms-loginmodal--default
2. Test in app:
  - Go to https://deploy-preview-2172--regen-marketplace.netlify.app/ and click login
  - Keplr/WC buttons behavior should be unchanged
  - Email login should send an email with magic link (but the rest of the flow is implemented separately: #2155)
  - Google login should be triggered as soon as https://github.com/regen-network/regen-server/pull/400 is merged so if that's not the case, please wait on that to be merged before testing this further
Also web2 based login will only reflect the current logged in user as soon as #2171 is done.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
